src/app/components/ContactToolbar.js

```diff
@@ -10,20 +10,20 @@ const ContactToolbar = ({
     onCheck,
     onDelete,
     checked = false,
-    activeIDs = [],
+    filteredCheckedIDs = [],
```

Comment:
What do you think about naming it `checkedIDs` instead? I'm not sure this component needs to be so specific in what is accepts. Perhaps it doesn't need to understand the concept of filtered? 

```suggestion
    checkedIDs = [],
```

Reply:
Yeah, I hesitated when writing it. I think you proposal makes sense :+1: